### ISSUE TITLE
Vue 3 Support?

### ISSUE THREAD

**Issue by calebeaires** [fgr-araujo/vue-shortkey#135]
Hi. I wonder when are you going to offer support to Vue 3. I had look at your code and seen that something can be done:

```js

ShortKey.install = (app, options) => {
    elementAvoided = [...(options && options.prevent ? options.prevent : [])];
    app.directive('shortkey', {
        created(el, binding, vnode) {
            // Mapping the commands
            const value = parseValue(binding.value);
            console.log(value);
            bindValue(value, el, binding, vnode);
        },
        updated(el, binding, vnode) {
            const oldValue = parseValue(binding.oldValue);
            unbindValue(oldValue, el);

            const newValue = parseValue(binding.value);
            bindValue(newValue, el, binding, vnode);
        },
        unmounted(el, binding) {
            const value = parseValue(binding.value);
            unbindValue(value, el);
        },
    });
};

```

This is the breaking changes from vue 2/3
https://v3.vuejs.org/guide/migration/custom-directives.html#custom-directives

**Comment by liquidvisual** [comment]
Would love to see Vue 3 support, this library is extremely useful.

**Comment by danielelkington** [comment]
Have raised a [pull request](https://github.com/iFgR/vue-shortkey/pull/142), and have also published an npm package [vue-three-shortkey](https://www.npmjs.com/package/vue-three-shortkey) to use in the meantime until the PR is merged.

**Comment by ghiscoding** [comment]
@danielelkington 
Adding your fork into my project and I can see that it also adds `custom-event-polyfill` in the lib's [package.json](https://github.com/danielelkington/vue-shortkey/blob/release/package.json) and I understand this polyfill was necessary in Vue 2 to support IE but in Vue 3 it is only meant to be used with greener browser so the polyfill could be dropped. It would be nice if you could do another update and drop that [line](https://github.com/danielelkington/vue-shortkey/blob/release/package.json#L18). Cheers and thanks for the quick temp fork to use.

**Comment by danielelkington** [comment]
Thanks for the tip @ghiscoding; have released [4.0.1](https://github.com/danielelkington/vue-shortkey/releases/tag/v4.0.1) of [vue-three-shortkey](https://www.npmjs.com/package/vue-three-shortkey) without this polyfill.

**Comment by killjoy1221** [comment]
I've found (when using vueuse), this package isn't strictly needed. Vueuse has a function named [`useMagicKeys`](https://vueuse.org/core/usemagickeys/) which returns an object of refs for each key combo, which can be watched.

Quick example of usage:

```js
import { watch } from "vue";
import { useMagicKeys } from "@vueuse/core";

const { escape } = useMagicKeys();

watch(escape, (v) => {
  if (v) {
    console.log("Escape was pressed");
  }
});
```
